src/models/sql/employees.model.js

```diff
@@ -78,15 +78,85 @@ const tableOptions = {
 
 const employees = connection.define(TABLE_NAME, fields, tableOptions);
 
-const save = async (dbDate) => {};
+const save = async (dbData) => {
+  dbData.employeeId = uuid.v4()
+  return employees.create(dbData);
+};
 
-const update = async (employeeId, dbData) => {};
+const update = async (req, res) => {
```

Comment:
Never pass req and res object to any other layer(service or model). Only pass the required data.
the controller will parse the request body and send it to service. The same way service will parse and construct the reqData for model and pass to model.

Reply:
Have modified the code as per your comments.

Reply:
ok